### PR TITLE
log-backup: update log backup default config value

### PR DESCRIPTION
**Pull request by Leavrth**
Signed-off-by: Leavrth <jianjun.liao@outlook.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/tikv/tikv/pull/13457

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-log-command-line.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ summary: 了解如何使用 br log 命令行工具进行日志备份并恢复日
 
 ### 启用日志备份
 
-使用日志备份功能前，需将 TiKV 配置项 [`log-backup.enable`](/tikv-configuration-file.md#enable-从-v620-版本开始引入) 设为 `true`。修改配置参数的方法，请参考[修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
+使用日志备份功能前，需将 TiKV 配置项 [`log-backup.enable`](/tikv-configuration-file.md#enable-从-v620-版本开始引入) 设为 `true`。目前该配置的默认值为 `true`。修改配置参数的方法，请参考[修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
```
Comment (shichun-0415) [inline br/br-log-command-line.md:21]:
```suggestion
使用日志备份功能前，需确保 TiKV 配置项 [`log-backup.enable`](/tikv-configuration-file.md#enable-从-v620-版本开始引入) 设为 `true`（默认值为 `true`）。修改配置参数的方法，请参考[修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
```

File: br/pitr-usage.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ summary: 了解如何使用 PITR。
 
 ## 启用日志备份
 
-使用日志备份功能前，需将 TiKV 配置项 [`log-backup.enable`](/tikv-configuration-file.md#enable-从-v620-版本开始引入) 设为 `true`。修改配置参数的方法，请参考[修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
+使用日志备份功能前，需将 TiKV 配置项 [`log-backup.enable`](/tikv-configuration-file.md#enable-从-v620-版本开始引入) 设为 `true`。目前该配置的默认值为 `true`。修改配置参数的方法，请参考[修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
```
Comment (shichun-0415) [inline br/pitr-usage.md:46]:
```suggestion
使用日志备份功能前，需确保 TiKV 配置项 [`log-backup.enable`](/tikv-configuration-file.md#enable-从-v620-版本开始引入) 设为 `true`（默认值为 `true`）。修改配置参数的方法，请参考[修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by joccau — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]

File: br/point-in-time-recovery.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ PITR 可用于满足以下业务需求：
 
 ### 版本兼容检查
 
-在 v6.3.0 中，PITR 产生的备份文件采用新的压缩方法，同时还会合并小文件（为解决之前小文件过多带来的问题），这会导致向前兼容性问题，详情如下表所示： 
+在 v6.3.0 中，PITR 产生的备份文件采用新的压缩方法，同时还会合并小文件（为解决之前小文件过多带来的问题），这会导致旧集群与新的备份数据不兼容的问题，详情如下表所示：
```
Comment (joccau) [inline br/point-in-time-recovery.md:97]:
```suggestion
在 v6.3.0 中，PITR 产生的备份文件采用新的压缩方法，同时还会合并小文件（为解决之前小文件过多带来的问题），这会导致旧版本 TiDB 集群与新版本产生的备份数据不兼容的问题，详情如下表所示： 
```

**Review by shichun-0415 — APPROVED** [review]

File: br/point-in-time-recovery.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ PITR 可用于满足以下业务需求：
 
 ### 版本兼容检查
 
-在 v6.3.0 中，PITR 产生的备份文件采用新的压缩方法，同时还会合并小文件（为解决之前小文件过多带来的问题），这会导致向前兼容性问题，详情如下表所示： 
+在 v6.3.0 中，PITR 产生的备份文件采用新的压缩方法，同时还会合并小文件（为解决之前小文件过多带来的问题），这会导致旧版本 TiDB 集群与新版本产生的备份数据不兼容的问题，详情如下表所示：
```
Comment (shichun-0415) [inline br/point-in-time-recovery.md:97]:
```suggestion
在 v6.3.0 中，PITR 产生的备份文件采用新的压缩方法，同时还会合并小文件（为解决之前小文件过多带来的问题），但是这也导致旧版本 TiDB 集群与新版本产生的备份数据不兼容，详情如下表所示： 
```

**Pull request merged**